app/controllers/ConfirmTransferController.scala

```diff
@@ -47,8 +48,8 @@ class ConfirmTransferController @Inject() (
   private def loadStandardPageBasedOnCtStatus(consignmentId: UUID, httpStatus: Status, finalTransferForm: Form[FinalTransferConfirmationData] = finalTransferConfirmationForm)(
       implicit request: Request[AnyContent]
   ): Future[Result] = {
-    consignmentStatusService.getConsignmentStatus(consignmentId, request.token.bearerAccessToken).flatMap { consignmentStatus =>
-      val exportTransferStatus = consignmentStatus.flatMap(_.export)
+    consignmentStatusService.getConsignmentStatuses(consignmentId, request.token.bearerAccessToken).flatMap { consignmentStatuses =>
+      val exportTransferStatus = consignmentStatusService.getStatusValues(consignmentStatuses, Export).values.head
```

Comment:
You've got quite a few of these `values.head` in a few different files but that will throw an exception if it's missing. I know it's unlikely but if you do `headOption.flatten` that should be ok.